src/test/java/updated/game/abstractClasses/Gamer.java

```diff
@@ -0,0 +1,43 @@
+package updated.game.abstractClasses;
+
+import lombok.Getter;
+import lombok.Setter;
+
+import java.util.ArrayList;
+import java.util.List;
+
+@Getter
+public abstract class Gamer {
+    protected String gamerName;
+    @Setter
+    public char symbol;
+    public int currentScore;
+    public int numberOfMoves;
+
```

Comment:
Can make currentScore and numberOfMoves private variables and make their getter and setter public methods.

Reply:
If i make it private, then i can't inherit them.

Reply:
Okay as you are using it to increment values it's fine. But it's better to make variables private and implementing their getters and setters as public method.